### PR TITLE
[15.0][MIG] helpdesk_type

### PR DESCRIPTION
**Pull request by dalonsod**
Standard migration + added test coverage

### CODE REVIEW

**Comment by dalonsod** [comment]
Hello @bodedra @DarioLodeiros could you review? Thanks!

File: helpdesk_type/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* Go to Helpdesk > Configuration > Types
```
Comment (bodedra) [inline helpdesk_type/readme/CONFIGURE.rst]:
```suggestion
* Go to Helpdesk > Configuration > Ticket Types
```

**Review by bodedra — APPROVED** [review]
Functional reviewed. 👍 

Added minor correction on menu navigation.

File: helpdesk_type/readme/CONFIGURE.rst (hunk 1 above)
Comment (dalonsod) [reply helpdesk_type/readme/CONFIGURE.rst]: Thanks, comment attended

**Comment by dalonsod** [comment]
@flachica could you review, please? Thanks!

**Review by flachica — APPROVED** [review]

**Comment by dalonsod** [comment]
@dreispt could you merge this PR? Thanks!

**Pull request merged**